SAFU.md

```diff
@@ -0,0 +1,78 @@
+- **Lack of clarity**: What to do with the funds secured from an exploit? Where should the white hat transfer the secured funds? Is there a compensation/reward promised for securing affected users’ funds?
+- **Execution risk**: Conflicting proposals and stressful negotiation may arise during and after the exploit of a vulnerability, leading to additional confusion and uncertainty between the parties involved.
+
+## Concepts
+
+- **Dropbox Address (”Dropbox”)**: An address or contract to which funds taken from the protocol should be deposited. In the case of contracts, a Dropbox can be *automatic*, handling claims and rewards on a per-depositor basis without human input, or *conditional*, requiring additional input such as governance approvals or identity verification (KYC).
+- **Deposit Interval:** the grace period in which a sender must deposit funds in the Dropbox after removing them from the protocol.
+- **Claim Delay:** a minimum waiting period before a sender may claim rewards. We recommend at least 24 hours, during which the extent of an exploit will become clear.
+- **Sender Claim Interval:** a maximum waiting period after which the protocol may reclaim the sender’s reward, to avoid leaving funds stranded in the contract.
+- B**ounty Percent**: Pro-rata share of funds secured that are claimable by the whitehats.
+- **Bounty Cap**: Maximum amount of tokens that a whitehat can claim after securing the vulnerable funds.
+
+## Statement for Whitehats
+
+Tharsis Labs Ltd. (the ”Team”) commits to not pursue legal action against white hats who act in accordance with the Arrangement for vulnerabilities found in the Evmos blockchain (chain ID: 9001).
+
+### Timeline
+
+The Team gives 48 hrs to hackers (”Grace Period”) to deposit the funds to the Dropbox from the moment they obtain the tokens from the exploited vulnerability. After this time, the Team will assume that the hacker is acting maliciously and against this Arrangement if they haven’t transferred the full amount of tokens.
+
+Evmos guarantees that the claiming for the funds process to begin not after 30 days (Claim Delay) ****of the transfer or during the next upgrade. This is due to the fact that transfers will need to be executed during an upgrade. We expect this process to become automatic after a dedicated trustless Cosmos module is incorporated for this purpose.
+
+If the whitehat doesn’t reclaim the tokens transferred before the 30th day from the transfer day (Sender **C**laim ****Interval**)**, the tokens will be reclaimed and transferred to the Evmos community pool (aka. community treasury).
+
+### Reward Policy
+
+Whitehats that secure vulnerable funds are is able to claim 5% of the total funds secured (Bounty Percent) up to a total of 750,000 EVMOS (Bounty Cap).
+
+There is no minimum to the amount that can be secured. The reward white hack hacker can secure from 1 atto EVMOS (1e-18 EVMOS or the equivalent unit of 1 wei on Ethereum).
```

Comment:
WDYM?

Reply:
This is the minimum amount that can be secured